soda-data-quality-checks/Makefile

```diff
@@ -0,0 +1,17 @@
+usage:		## Shows usage for this Makefile
+	@cat Makefile | grep -E '^[a-zA-Z_-]+:.*?## .*$$' | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}'
+
+install:    ## Install dependencies
```

Comment:
Could be out of the scope of this PR, but maybe we can make sure package installation happens inside users virtual environment?